docs/modules/ROOT/pages/tutorials/migration.adoc

```diff
@@ -0,0 +1,619 @@
+
+== Let's Encrypt
+
+In APPUiO Cloud only `Ingress` objects can request certificates.
+
+IMPORTANT: It's not possible to request certificates in `Route` objects.
+
+The manifest below specifies an `Ingress` object with such requirement:
+
+[source,yaml]
+--
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-production
+    kubernetes.io/ingress.class: nginx
+    kubernetes.io/tls-acme: "true" # <1>
+  name: ${DEPLOYMENT_NAME}
+  namespace: oauth-test
+spec:
+  rules:
+  - host: ${ROUTE_HOST}
+    http:
+      paths:
+      - backend:
+          service:
+            name: nginx
+            port:
+              number: 80
+        path: /
+        pathType: ImplementationSpecific
+  tls:
+  - hosts:
+    - ${ROUTE_HOST}
+    secretName: ${ROUTE_HOST}
+--
+<1> Annotation providing automatic provisioning of certificates on APPUiO Cloud.
+
+You can find more information in the xref:how-to/getting-a-certificate.adoc[Getting a Certificate through Let’s Encrypt] page.
+
+For reference, it was possible to configure TLS certificates in `Route` objects in APPUiO Public, using a manifest similar to the following. This is no longer valid in APPUiO Cloud.
+
+[source,yaml]
+--
+- apiVersion: route.openshift.io/v1
+  kind: Route
+  metadata:
+    name: ${DEPLOYMENT_NAME}
+    labels:
+      app: ${CI_ENVIRONMENT_SLUG}
+    annotations:
+      kubernetes.io/tls-acme: "${ROUTE_TLS_ACME}" # <1>
+  spec:
+    host: ${ROUTE_HOST}
+    port:
+      targetPort: http
+    tls:
+      insecureEdgeTerminationPolicy: Redirect
+      termination: edge
+    to:
+        kind: Service
+        name: ${DEPLOYMENT_NAME}
+        weight: 100
+    wildcardPolicy: None
+--
+<1> Annotation providing automatic provisioning of certificates on APPUiO Public.
+
+== Storage
+
+APPUiO Cloud has different requirements regarding storage. This section provides more information.
+
+=== Access the RWO volumes from different pods
+
+When migrating your volumes to APPUiO Cloud, please follow these guidelines:
+
+. The rollout strategy must change from `RollingUpdate` to `Recreate`, because mounting the volume at different nodes simultaneously isn't possible, or at least not with common used filesystems, such as https://en.wikipedia.org/wiki/Ext4[ext4] and https://en.wikipedia.org/wiki/XFS[XFS]. https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy[This page] in the Kubernetes documentation provides more information about this.
+
+. Add an affinity rule to run on the same node as the worker pod. This allows to access the same RWO/block volume, because the filesystem is mounted on the node level, and allows a safe access to the same filesystem, because it's the same kernel.
```

Comment:
```suggestion
. Add an affinity rule to run on the same node as the worker pod. 
  This allows to access the same RWO/block volume, because the filesystem is mounted on the node level, and allows a safe access to the same filesystem, because it's the same kernel.
```